lib/takeaway.rb

```diff
@@ -0,0 +1,65 @@
+class Takeaway
+  
+  attr_reader :menu, :basket
+  DEFAULT = 1
+
+  def initialize
+    @menu = { 'Fish' => 1, 
+             'Burger' => 2, 
+             'Chips' => 3, 
+             'Pizza' => 8, 
+             'Pasta' => 0.5
+            }
+    @basket = []
```

Comment:
This is your order, there is an opportunity here to use single reposibility principle and extract a class Basket and add methods pertaining to this Basket into it